solidity/random-beacon/contracts/test/StakingStub.sol

```diff
@@ -8,6 +8,8 @@ import "../RandomBeacon.sol";
 contract StakingStub is IRandomBeaconStaking {
     mapping(address => uint256) public stakedTokens;
 
+    bool public shouldFail;
```

Comment:
I think we might need to remove these tests once we integrate T staking contract 😬 But that's fine, let's leave them for now and see what can be done in the future.